app/docker/components/datatables/tasks-datatable/tasksDatatable.js

```diff
@@ -1,6 +1,6 @@
 angular.module('portainer.docker').component('tasksDatatable', {
   templateUrl: 'app/docker/components/datatables/tasks-datatable/tasksDatatable.html',
-  controller: 'GenericDatatableController',
+  controller: 'TasksDatatableController',
```

Comment:
Note that the introduction of this Controller breaks some table functionalities (column sorting, pagination). You'll probably need to rewrite the `TasksDatatableController`.

Reply:
Any tipp to fix this without copy the complete `GenericDatatableController`?

Reply:
@farodin91 not really, you'll have to copy the GenericDatatableController and introduce the new state object.